docs/iam/index.md

```diff
@@ -3,19 +3,30 @@ title: Identitets og tilgangsstyring (Felles IAM)
 disableTitleTagline: true
 ---
 
-Identitets- og tilgangsstyring (IAM) handler om å gi studenter, forskere og ansatte riktig tilgang til digitale systemer, tjenester og ressurser, samt avslutte tilgang for brukere som ikke lenger skal ha det.
+Identitets- og tilgangsstyring (Felles IAM) handler om å gi studenter, ansatte og gjester de riktige tilgangene til digitale systemer og tjenester. Samt avslutte tilganger til systemer, tjenester og ressurser for brukere som ikke lenger skal ha det.
 
-[Les mer om IAM-tjenesten på sikt.no](https://sikt.no/tjenester/felles-iam)
+Basisproduktet som benyttes i denne Felles IAM løsningen er [Rapid Identity](./produkt), utviklet av tredjepartsleverandøren Identity Automations. Sikt leverer tilpasninger og implementering i Norge i henhold til UH sektoren og virksomheter behov og endringsønsker. I tillegg benyttes Account Claim (kontoaktivering) som Sikt både utvikler og drifter.
 
-Her vil du finne overordnet systemdokumentasjon, og etterhvert brukerdokumentasjon. [En overordnet arkitektur for IAM](./arkitektur) og [gevinster](./gevinster) vil være en gode steder å starte for å bedre forstå IAM.
+For en overordnet beskrivelse av funksjonalitet [Les mer om IAM-tjenesten på sikt.no](https://sikt.no/tjenester/felles-iam)
 
-Felles IAM vil [livssyklusen til studenter, ansatte og gjester](./livssyklus), og opprette, endre og fjerne digitale kontoer basert på data fra [kildesystemer](./kildedata). Felles IAM kan identifisere en person på tvers av utdanningsinstitusjoner, og [generere et nasjonalt unikt brukernavn og en identifikator](./brukernavn). Når en bruker møter en utdanningsinstitusjon for første gang vil vedkommende møte [en brukervennlig kontoaktivering](./kontoaktivering), hvor man blant annet setter passordet, i henhold til [felles passordpolicy](./passordpolicy). Noe av den mest sentrale funksjonaliteten i Felles IAM er [tilgangsstyring til målsystemer](./tilgangsstyring), hvor tilganger kan settes basert på [virksomhetsroller](./virksomhetsroller).
+For informasjon om arkitektur se
+[En overordnet arkitektur for IAM](./arkitektur) 
```

Comment:
Kanskje slå sammen disse to linjene så ligner de mer på linje 15.

Reply:
Done